### PR TITLE
Added tests for the user router

### PR DESCRIPTION
**Pull request by TheRealSeber**

### CODE REVIEW

File: app/api/tests/test_user.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    assert response.status_code == 200
+    assert "id" in response.json()
+
+
+def test_user_patch(client):
+    test_user = user_schemas.UserPut(
+        name="test_name", surname="test_surname", email="test_email@abc.com"
+    )
+    response = client.put(ROUTE, json=dict(test_user))
+    assert response.status_code == 200
```
Comment (matined) [inline app/api/tests/test_user.py:43]: again, do not test two functionalities in one unit test

File: app/api/tests/test_user.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    user_id = response.json()["id"]
+
+    test_user_patch = user_schemas.UserPatch(
+        name="test_patch_name",
+        surname="test_patch_surname",
+        email="test_patch_email@abc.com",
+    )
+    response = client.patch(f"{ROUTE}{user_id}", json=dict(test_user_patch))
+    assert response.status_code == 200
```
Comment (matined) [inline app/api/tests/test_user.py:52]: put all asserts together at the end of a test and leave an empty line before them

File: app/api/tests/test_user.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    data = response.json()
+    assert data == {
+        "id": user_id,
+        "name": "test_patch_name",
+        "surname": "test_patch_surname",
```
Comment (matined) [inline app/api/tests/test_user.py:57]: put excepted data before assert and at the put just

assert response.status_code == expected_status_code
assert data == expected_data

**Review by matined — CHANGES_REQUESTED** [review]
this comments apply to all the tests

File: .flake8 [hunk 4]
```diff
@@ -1,3 +1,5 @@
 [flake8]
 max-line-length = 88
-extend-ignore = E203
\ No newline at end of file
+extend-ignore = E203
```
Comment (matined) [inline .flake8:3]: why do we add E203

**Review by matined — APPROVED** [review]

File: .flake8 (hunk 4 above)
Comment (TheRealSeber) [reply .flake8:3]: It has already been there, for some reason adding  two new lines marked it as if it was deleted and then added again

Comment (matined) [reply .flake8:3]: ok

**Pull request merged**